pkg/util/httputil/permissions_policy.go

```diff
@@ -0,0 +1,117 @@
+package httputil
+
+import (
+	"fmt"
+	"strings"
+)
+
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy#directives
+type PermissionsPolicyDirective string
+
+const (
+	PermissionsPolicyDirectiveAccelerometer               PermissionsPolicyDirective = "accelerometer"
+	PermissionsPolicyDirectiveAmbientLightSensor          PermissionsPolicyDirective = "ambient-light-sensor"
+	PermissionsPolicyDirectiveAutoplay                    PermissionsPolicyDirective = "autoplay"
+	PermissionsPolicyDirectiveBattery                     PermissionsPolicyDirective = "battery"
+	PermissionsPolicyDirectiveBluetooth                   PermissionsPolicyDirective = "bluetooth"
+	PermissionsPolicyDirectiveCamera                      PermissionsPolicyDirective = "camera"
+	PermissionsPolicyDirectiveDisplayCapture              PermissionsPolicyDirective = "display-capture"
+	PermissionsPolicyDirectiveDocumentDomain              PermissionsPolicyDirective = "document-domain"
+	PermissionsPolicyDirectiveEncryptedMedia              PermissionsPolicyDirective = "encrypted-media"
+	PermissionsPolicyDirectiveExecutionWhileNotRendered   PermissionsPolicyDirective = "execution-while-not-rendered"
+	PermissionsPolicyDirectiveExecutionWhileOutOfViewport PermissionsPolicyDirective = "execution-while-out-of-viewport"
+	PermissionsPolicyDirectiveFullscreen                  PermissionsPolicyDirective = "fullscreen"
```

Comment:
Missing `gamepad` after `fullscreen`

Reply:
Updated, should have 36 items now matching doc's list 🙏